app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -103,6 +114,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_attendees, container, false);
 
         binding.setAttendees(attendeesPresenter.getAttendees());
+        setRetainInstance(true);
```

Comment:
Please remove this. It is causing issue while screen rotation
On screen rotation all loaded data in the layout vanishes.